modules/schematics/src/reducer/files/__name@dasherize@if-flat__/__name@dasherize__.reducer.ts

```diff
@@ -14,8 +14,13 @@ export function reducer(state = initialState, action: <% if(feature) { %><%= cla
 <% if(feature) { %>
     case <%= classify(name) %>ActionTypes.Load<%= classify(name) %>s:
       return state;
+<% if(api) { %>
```

Comment:
Could you remove the import `import { Action } from '@ngrx/store';`, it isn't in use?

Reply:
I'll add conditional logic to only import this if `!feature`

Reply:
This should be resolved in the latest commit.